local-volume/provisioner/pkg/common/common.go

```diff
@@ -149,8 +149,7 @@ func CreateLocalPVSpec(config *LocalPVConfig) *v1.PersistentVolume {
 			Name:   config.Name,
 			Labels: config.Labels,
 			Annotations: map[string]string{
-				AnnProvisionedBy:                      config.ProvisionerName,
-				v1.AlphaStorageNodeAffinityAnnotation: config.AffinityAnn,
```

Comment:
I think instead of completely removing, it should be controlled by a configuration flag, with beta as the default.

Reply:
yes, i see. 
and as you suggested, i splited this task into several sub tasks,  this PR is just to upgrade kubernetes dependencies

Reply:
see: https://github.com/kubernetes-incubator/external-storage/issues/638